hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -75,14 +73,6 @@
  * @see HoodieTableMetaClient
  * @since 0.3.0
  */
-@ConfigClassProperty(name = "Table Configurations",
```

Comment:
Removing annotation for `HoodieTableConfig` so they do not show up on the configuration page, as these are not meant to be changed by the user.